src/pages/data-management/index.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-unresolved */
```

Comment:
why are you adding this?

Reply:
because we can import stuff without a relative path but like 'utils/something.js' eslint still marks it as unresolved though.

Reply:
This is because we added `baseUrl` in `jsconfig.json` : https://nextjs.org/docs/advanced-features/module-path-aliases, but it's not yet widely used it seems

Reply:
what is it about? If it is not used, should we remove it?

Reply:
I'm using it in some places i find it convenient to import things without gettting lost in the path